src/Request.php

```diff
@@ -7,22 +14,48 @@
  */
 class Request
 {
+    /**
+     * @var string
+     */
     private $url;
+
+    /**
+     * @var string
+     */
     private $content;
+
+    /**
+     * @var array
+     */
     private $headers = [];
 
+    /**
+     * @param string $url
+     * @param string $content
+     * @param array  $headers
+     */
     public function __construct($url, $content = '', $headers = [])
     {
-        $this->url = $url;
+        $this->url     = $url;
         $this->content = $content;
         $this->headers = $headers;
     }
 
+    /**
+     * @param  string $url
+     * @param  string $content
+     * @param  array  $headers
```

Comment:
I would separate the params from the return value rather than aligning them together (same everywhere)